### ISSUE TITLE
Missing results (stress, strain etc) on beams

### ISSUE THREAD

**Issue by FedericoNegri** [ansys/pydpf-post#711]
### Before submitting the issue

- [X] I have checked for [Compatibility](https://dpf.docs.pyansys.com/version/stable/getting_started/index.html#compatibility) issues
- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### Description of the bug

Stress (as well as other) results are not extracted on beam elements. 

### Steps To Reproduce

```python
from ansys.dpf import post

file_path = "post_beam_result_01.rst"

simulation = post.StaticMechanicalSimulation(file_path)
seqv = simulation.stress_eqv_von_mises_nodal()
print(seqv._fc[0])
```

output

```
DPF stress_1.s_eqv Field
  Location: Nodal
  Unit: in^-1*slinch*s^-2
  0 entities
  Data: 1 components and 0 elementary data
```

RST file to run the script: [post_beam_result_01.zip](https://github.com/user-attachments/files/17190638/post_beam_result_01.zip)

### Which Operating System are you using?

Windows

### Which DPF/Ansys version are you using?

DPF Server 2025.1.pre0

### Which Python version are you using?

3.10

### Installed packages

--

**Comment by FedericoNegri** [comment]
@janvonrickenbach I guess the `read_beams` pin need to be set for the result operators supporting it?

**Comment by janvonrickenbach** [comment]
@FedericoNegri Is it really expected that we have stress results for beams? With #778 displacement results are available for the full model as well as on the skin.

**Comment by FedericoNegri** [comment]
Yeah I think we'd anyway need special treatment for beam stresses, either show it separately or do beam expansion. Closing this one.

**Comment by janvonrickenbach** [comment]
I'm not aware of any issue / user story. Maybe @rafacanton knows more.
